src/main/java/org/kitteh/irc/client/library/Client.java

```diff
@@ -829,4 +832,14 @@ default void sendMultiLineNotice(@Nonnull MessageReceiver target, @Nonnull Strin
      * @param reason quit message to send
      */
     void shutdown(@Nonnull String reason);
+
+    /**
+     * Sends a DCC CHAT request to the target.
+     *
+     * <p>When the server socket is bound locally, a
+     * {@link DCCSocketBoundEvent} will be fired. When the chat is connected,
+     * a {@link DCCConnectedEvent} will be fired. If the connection fails, a
+     * {@link DCCFailedEvent} will be fired.</p>
+     */
+    void requestDCCChat(@Nonnull User target);
```

Comment:
I don't know if that should be needed? All DCC chats will come through as events so you can keep tabs on them if you want.

However, it currently isn't possible to tell if you have multiple DCC exchanges open to a client or distinguish between them, so maybe we can figure out a way to do that and add a way to get a specific event?